hooks/ad-join-relation-joined.ps1

```diff
@@ -22,27 +22,28 @@ try {
     Import-Module JujuHooks
     Import-Module JujuUtils
 
+    $renameReboot = Rename-JujuUnit
```

Comment:
I just bootstrapped a Juju controller on OpenStack with latest stable Juju 2.2.1 and nova instances spawned by Juju still have a common preffix longer than 15 characters (see [here](http://paste.ubuntu.com/24933180/)). So, we still need this `Rename-JujuUnit`.